tox.ini

```diff
@@ -30,15 +31,25 @@ extras =
 [testenv:lint]
 basepython = python3
 skip_install = true
+commands =
+    isort --check-only --diff {toxinidir}/src {toxinidir}/setup.py
+    flake8 src setup.py
+    check-manifest
+    check-python-versions
 deps =
-    flake8
     check-manifest
     check-python-versions >= 0.19.1
     wheel
+    flake8
+    isort
+
+[testenv:isort-apply]
+basepython = python3
+commands_pre =
+deps =
+    isort
```

Comment:
This (and other tool-oriented testenvs) should have a `skip_install = true` because we don't need to pip install zope.testrunner to run isort/flake8/etc.

This is more of a self-comment to go and file the same issue against the meta/config repo.

Reply:
Fixed in https://github.com/zopefoundation/zope.testrunner/pull/140/commits/b334aff904d33fda4d8e5a110a0288c81d1680ca